api/liboni/drivers/oepcie/liboni-driver-oepcie.c

```diff
@@ -0,0 +1,362 @@
+#include <assert.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "liboni-driver-oepcie.h"
+#include "../../oni-driver.h"
+
+#ifdef _WIN32
+#include <io.h>
+#define open _open
+#define read _read
+#define write _write
+#define close _close
+#define lseek _lseek
+#pragma intrinsic (_InterlockedIncrement)
```

Comment:
remove unnessesary CPP stuff (e.g. Interlocked...)

Reply:
Those were in the original oni.c. I didn't know if there were there for some obscure performance optimization. I'll remove them then